### PR TITLE
IITC-Mobile: fix bugs on login page

### LINKED ISSUE #154: Google authorization does't work on Android KitKat
The authorization form is not displayed correctly, data cannot be sent.

Temporary solution:
Enable *Fake User Agent* in *Settings → Advanced Settings*.

Explanation:
This results to loading of another authorization form that runs on an outdated KitKat browser engine.

### LINKED ISSUE #160: IITCm signin layout
After recent intel update there is new button which does not fit well:
<details>

IITC screenshot-1288478869

</details>

### PR DESCRIPTION
**Pull request by modos189**
Fix sign-in buttons layout, which was broken after Niantic added facebook authorization, and more.
Fix #160, fix #154, fix (part of) https://github.com/IITC-CE/ingress-intel-total-conversion/issues/163#issuecomment-482516494

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ document.body.onload = function() {};
 //originally code here parsed the <Script> tags from the page to find the one that defined the PLAYER object
 //however, that's already been executed, so we can just access PLAYER - no messing around needed!
 
-var PLAYER = window.PLAYER || (unsafeWindow && unsafeWindow.PLAYER);
+var PLAYER = window.PLAYER || (typeof unsafeWindow !== 'undefined' && unsafeWindow && unsafeWindow.PLAYER);
```
Comment (johnd0e) [inline main.js]: The matter of taste, but I still think that one condition is excessive
```suggestion
var PLAYER = window.PLAYER || (typeof unsafeWindow !== 'undefined' &&  unsafeWindow.PLAYER);
```

File: main.js [hunk 2]
```diff
@@ -37,6 +37,12 @@ if (typeof(PLAYER)!="object" || typeof(PLAYER.nickname) != "string") {
     throw("Page doesn't have player data, but you are logged in.");
   }
   // FIXME: handle nia takedown in progress
+  
+  // add login form stylesheet
```
Comment (johnd0e) [inline main.js:41]: Should we change style for desktop IITC?

File: main.js (hunk 1 above)
Comment (modos189) [reply main.js]: what if `typeof unsafeWindow !== 'object'`,

File: main.js (hunk 2 above)
Comment (modos189) [reply main.js:41]: Well, why not? CSS `media` can create an adaptive view for all screen resolutions

File: main.js (hunk 1 above)
Comment (johnd0e) [reply main.js]: Why? Can you imaging real case?

File: main.js (hunk 2 above)
Comment (johnd0e) [reply main.js:41]: Ok. I have no objections (though not seen new design yet).

File: main.js (hunk 1 above)
Comment (modos189) [reply main.js]: Oh. `===`, not `!==`.
But I agree, it makes no sense, because in JavaScript everything is an objects.

**Comment by johnd0e** [comment]
@modos189 

<details><summary>width >=970 (responsive)</summary>

image
</details>

<details><summary>width: <970 (too wide)</summary>

image
</details>

<details><summary>width: <969 (unresponsive)</summary>

image
image

</details>

Caused by 
```css

#dashboard_container {
    min-height: 500px;
    min-width: 930px;
}
```

I suppose we should allow lower values, say, `min-width: 245px; min-height: 365px`.

**Comment by modos189** [comment]
on mobile:

<details>
<summary>before #f222ab6</summary>

изображение

</details>


<details>
<summary>after #f222ab6</summary>

изображение


</details>

I suggest add

```
@media (max-width: 970px) { 		
    .button { 		
        margin: 5px 5%;	
    } 		
}
```

<details>
<summary>then in look like this</summary>

изображение


</details>

**Comment by johnd0e** [comment]
> on mobile:

I do not see much difference on mobile (both variants seem acceptable).
But at desktop I do not like too wide buttons with small margins.

**Comment by modos189** [comment]
Ok

**Pull request merged**